crates/trie/sparse/src/metrics.rs

```diff
@@ -16,24 +17,28 @@ pub(crate) struct SparseStateTrieMetrics {
     /// Number of total storage nodes, including those that were skipped.
     pub(crate) multiproof_total_storage_nodes: u64,
```

Comment:
the way this is structured feels a bit strange, ideally this entire file is feature gated but this would require rewriting how metrics are recorded and we can do that separately.

Reply:
Yeah that was my initial thought too